docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,165 @@
+
+In this guide, we will migrate from the Expo Firebase packages to React Native Firebase by:
+- Replacing the Expo Firebase packages with their React Native Firebase equivalents
+- Replacing the use of Expo Go for testing with a [development build](/development/introduction/) customized to your project's needs
+- Testing the updated code on your development build
+
+> **info** These instructions assume you have already been successfully using the Expo Firebase packages. If you're adding Firebase to your app for the first time, see the [Using Firebase](/guides/using-firebase) guide.
+
+## Install React Native Firebase
+Run the following commands to remove `expo-firebase-analytics` and install the React Native Firebase packages:
+
+<Terminal cmd={[
+'# Remove Expo Firebase packages',
+'$ yarn remove expo-firebase-analytics',
+'',
+'# Install RN Firebase, etc.',
+'$ yarn add @react-native-firebase/app @react-native-firebase/analytics'
+]} />
+
+If you are using [Expo Prebuild](/guides/adopting-prebuild) (also known as Managed Workflow) and building your app for iOS, install the `expo-build-properties` plugin and apply the following configuration to your [Expo config](/workflow/configuration/) file.
+
+<Terminal cmd={["$ npx expo install expo-build-properties"]} />
+
+### Add Config plugin to Expo config file
+
+In the Expo config file, add add `expo-community-flipper` to the `plugins` array with the following configuration:
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-build-properties",
+        {
+          "ios": {
+            "useFrameworks": "static"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+If you are using the Bare Workflow, see [React Native Firebase's installation instructions](https://rnfirebase.io/#bare-workflow).
+
+## Update analytics requests
+
+Firebase Analytics requests in React Native Firebase are almost the same - mostly, they are imported a little differently. Update your code as shown here:
+
+```diff
+// update imports
+- import * as Analytics from "expo-firebase-analytics";
++ import analytics from "@react-native-firebase/analytics";
+
+// update each request to analytics
+- Analytics.logEvent("test_analytics_event", {
+-  additionaParam: "test",
+- });
++ analytics().logEvent("test_analytics_event", {
++  additionaParam: "test",
++ });
+```
+
+`logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
+
+## Creating a development build
+
+If you have previously used Expo Go for development and testing, at this point, you will need to migrate to a [development build](/development/introduction/#from-expo-go-to-development-builds), as Expo Go does not include the native code used in React Native Firebase.
+
+[Follow these instructions for creating a development build](/development/getting-started/). Once you are finished, you should have an **eas.json** file that includes a build profile that looks like this:
```

Comment:
we should delegate this section to a guide about migrating from expo go to development builds so we don't repeat ourselves (i know you tried to avoid doing that too much here but i think it's better to not even try to summarize)